src/test/java/com/algorand/algosdk/integration/Clients.java

```diff
@@ -9,10 +9,19 @@
 
 public class Clients {
     AlgodClient v2Client = null;
+
+    @Deprecated
     Map<Integer, IndexerClient> indexerClients = new HashMap<>();
```

Comment:
shall I just remove this stuff? It should not get into SDK interface tho.

Reply:
Yeah, I recommend removing completely. We're only going to have one indexer. I missed this in a previous PR.